src/TractionRecClient.php

```diff
@@ -138,11 +138,18 @@ public function getAccessToken(): string {
     }
     catch (RequestException $e) {
       $response = $e->getResponse();
+      $this->logger->error($e->getMessage());
```

Comment:
Let's add prefix for error to be well visible in external logging systems
```suggestion
      $this->logger->error('Traction Rec Request Exception: ' . $e->getMessage());
```

Reply:
Line 105
`$this->logger = $logger_channel_factory->get('traction_rec');`

![SCR-20240208-jjja](https://github.com/YCloudYUSA/openy_traction_rec/assets/238201/b5f026a9-ee7e-4746-b2ba-db451329462a)

Logs are categorized already, so I don't think we need the additional text description.